src/constructs/EcsFargateService.ts

```diff
@@ -146,7 +146,27 @@ export class EcsFargateService extends Construct {
       executionRole: this.setupTaskExecutionRole(),
     });
 
-    taskDef.addContainer(`${props.serviceName}-container`, {
+    // Create a volume for the container to write to
+    const VOLUME_NAME = 'issue-container-volume';
+    taskDef.addVolume({ name: VOLUME_NAME });
+
+    // The init container will change permissions on the volume
+    // See https://github.com/aws/containers-roadmap/issues/938
+    const initContainer = taskDef.addContainer('init-container', {
+      image: ecs.ContainerImage.fromRegistry('alpine:latest'),
+      entryPoint: ['sh', '-c'],
+      command: ['chmod 0777 /storage'], // TODO check proper restriction
```

Comment:
Hebben we een issue om te tracks dat deze niet op 0777 blijft staan? Zo ja, prima!